changelog.json

```diff
@@ -1,5 +1,23 @@
 {
   "releases": {
+    "3.2.5": [
+      "[Fixed] Entering in double forward slash does not  target directory in cloning dialog - #15842. Thanks @IgnazioGul!",
```

Comment:
Nit & TOC 😂 
```suggestion
      "[Fixed] Entering in double forward slash does not target directory in cloning dialog - #15842. Thanks @IgnazioGul!",
```

Reply:
Actually I think this is missing a word...